tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/work_order_wrap.cpp

```diff
@@ -38,15 +36,16 @@ std::string HandleWorkOrderRequest(
     tcf::enclave_queue::ReadyEnclave readyEnclave = \
         tcf::enclave_api::base::GetReadyEnclave();
 
-    presult = tcf::enclave_api::workorder::HandleWorkOrderRequest(
+    WorkOrderHandlerSingleton wo_handle;
```

Comment:
Is this function applicable for Singleton. KME and WPE.  If so, how is this going to differentiate between  multiple enclave types ?

Reply:
This function is specific to Singleton. I have created similar wrapper files for KME and WPE and will be raising PR for them post this.